repos/dir.targets

```diff
@@ -34,6 +34,7 @@
     <DependencyVersionInputRepoApiImplemented Condition="'$(DependencyVersionInputRepoApiImplemented)' == ''">$(RepoApiImplemented)</DependencyVersionInputRepoApiImplemented>
     <SourceOverrideRepoApiImplemented Condition="'$(SourceOverrideRepoApiImplemented)' == ''">$(RepoApiImplemented)</SourceOverrideRepoApiImplemented>
     <OutputPlacementRepoApiImplemented Condition="'$(OutputPlacementRepoApiImplemented)' == ''">$(RepoApiImplemented)</OutputPlacementRepoApiImplemented>
+    <LogVerbosityRepoApiImplemented Condition="'$(LogVerbosityRepoApiImplemented)' == ''">$(RepoApiImplemented)</LogVerbosityRepoApiImplemented>
```

Comment:
I think I was overthinking this... there are potentially issues with adding this as a repo API arg (details below) and now I think it makes the most sense to put this back in the Exec and call it good.

Looking at `BuildPackagesCommand` and how `/v` is in it directly is what tipped this for me. There's no generalized arg property to move *that* `/v` into, so treating this as a repo API arg actually seems a little inconsistent. My bad.

---

I'm pretty sure this actually results in the arg being applied in different places than your original PR... a lot of projects have `RepoApiImplemented` set to `false`, but the intent is to pass this into those cases anyway.

I still think it makes the most sense to call this a repo API argument--I see two ways to resolve this:

1. Add `<LogVerbosityRepoApiImplemented>true</LogVerbosityRepoApiImplemented>` to every repo that has `RepoApiImplemented` `false` and specifies a build command.

2. Make this a specific opt-out separate from `RepoApiImplemented`, like:

    ```suggestion
        <LogVerbosityRepoApiImplemented Condition="'$(LogVerbosityRepoApiImplemented)' == ''">true</LogVerbosityRepoApiImplemented>
    ```

    And a comment like `Even if a repo doesn't implement the general repo API, it probably handles the MSBuild verbosity argument. Make verbosity a specific opt-out option.`

2 is a more minimal change, and I'm good with it. If more people had to interact with these flags I'd be more worried about keeping `RepoApiImplemented` "pure".

Reply:
"I think it makes the most sense to put this back in the Exec and call it good" vs "I still think it makes the most sense to call this a repo API argument" is making me a little confused. I am happy to do what you suggest is the best option, but can you clarify what style you would prefer?

Reply:
Yeah... when I wrote that comment, at first I wanted to stick with repo API args, and the bottom section talks about that. Then I reconsidered and wrote the top part. Definitely isn't clear, sorry about that.

I think going back to including the arg in the Exec is best for this PR.

Reply:
@dagood Does this look like what you were thinking?  (@omajid, hope you don't mind, I rebased and did a little cleanup)

Reply:
I'd expect to see this change back in `RepoBuild`, similar to how it is in `Package`:

```diff
-    <Exec Command="$(BuildCommand) $(RepoApiArgs) $(RedirectRepoOutputToLog)" WorkingDirectory="$(ProjectDirectory)" EnvironmentVariables="@(EnvironmentVariables)" />
+    <Exec Command="$(BuildCommand) $(RepoApiArgs) /v:$(LogVerbosity) $(RedirectRepoOutputToLog)" WorkingDirectory="$(ProjectDirectory)" EnvironmentVariables="@(EnvironmentVariables)" />
```

Basically, any repo that overrides `RepoBuild` has to do this themselves, but if they use `RepoBuild`, they'll just get the parameter automatically.

Reply:
Hmm, actually I'm not sure all our `BuildCommand`s are guaranteed to accept the `/v` argument (since some of them are custom build scripts).  I'll give it a try.